pages.zh/linux/debootstrap.md

```diff
@@ -0,0 +1,20 @@
+# debootstrap
+
+> 创建一个基本的Debian系统。
+> 更多信息： <https://wiki.debian.org/Debootstrap>。
```

Comment:
I'm not sure if there's supposed to a be space after the full-width colon — am I correct about that?
```suggestion
> 更多信息：<https://wiki.debian.org/Debootstrap>。
```